modules/payloads/singles/cmd/unix/reverse_jjs.rb

```diff
@@ -36,6 +36,11 @@ def initialize(info = {})
     register_options [
       OptString.new('SHELL', [ true, 'The shell to execute.', '/bin/sh' ])
     ]
+    register_advanced_options(
+      [
+        OptString.new('JJSPath', [true, 'The Path to JJS executable', 'jjs'])
```

Comment:
```suggestion
        OptString.new('JJSPath', [true, 'The path to the JJS executable', 'jjs'])
```

This applies to all other files. `Path` should be lowercase, and your missing a `the` after the `to`.

Reply:
Confirmed resolved.